README.md

```diff
@@ -50,14 +40,32 @@ please use the [GSA/code-gov](https://github.com/GSA/code-gov) repo as a catch a
+| [GSA/code-gov-github-metrics](https://github.com/GSA/code-gov-github-metrics)                       | This project compiles and calculates GitHub metrics across the different repos that make up code.gov so that the code.gov team can understand and track community contributions and other data points over time. | [View all issues](https://github.com/GSA/code-gov-github-metrics/issues)            | [Create a new issue](https://github.com/GSA/code-gov-github-metrics/issues/new)            |
+| [GSA/code-gov-open-source-toolkit](https://github.com/GSA/code-gov-open-source-toolkit)                       | This is a government-wide project facilitated by the Code.gov team to produce a playbook and toolkit pertaining to open sourcing software (OSS). | [View all issues](https://github.com/GSA/code-gov-open-source-toolkit/issues)            | [Create a new issue](https://github.com/GSA/code-gov-open-source-toolkit/issues/new)            |
+
+### Deprecated repositories
+
+These repos were previously used by the code.gov team but have been deprecated and are no longer in use
+
+| Project                                                                       | Description | Reason for deprecation|
+| ----------------------------------------------------------------------------- | ------ | ------------------------------------------------------------------------------- |
+| [GSA/code-gov-web](https://github.com/GSA/code-gov-web)                               | The old version of the [Code.gov](https://code.gov) front end | The front end stack was changed from Angular to React. The new front end repo is [GSA/code-gov-front-end](https://github.com/GSA/code-gov-front-end). |
+| [GSA/code-gov-harvester-deprecated](https://github.com/GSA/code-gov-harvester-deprecated)                               | The old version of the harvester that harvests and processes code.json files from agencies. | The new harvester repo is [GSA/code-gov-harvester](https://github.com/GSA/code-gov-harvester) |
+| [GSA/code-gov-about-page](https://github.com/GSA/code-gov-about-page)                               | Component for the About page on [Code.gov](https://code.gov). | The about page component was integrated directly into [GSA/code-gov-front-end](https://github.com/GSA/code-gov-front-end) in order to simplify the site architecture |
+| [GSA/code-gov-fscp-react-component](https://github.com/GSA/code-gov-fscp-react-component)                               | Federal Source Code Policy (FSCP) plugin for code-gov-web, built with Markdown and React. | The FSCP is hosted at [https://sourcecode.cio.gov/](https://sourcecode.cio.gov/) and it was therefore duplicative for it to be on [Code.gov](https://code.gov) as well |
```

Comment:
Minor correction. code-gov-fscp-react-component was originally built for the React version of code.gov.  If my memory serves me correctly, when I originally created code-gov-fscp-react-component, the React version hadn't been named `code-gov-front-end` yet.  Once a naming decision was made, I forgot to update the description in the plugin repo Readme.  So this should read `... (FSCP) plugin for code-gov-front-end, built with Markdown and React`.  I'm sorry for the confusion.

Reply:
Ah ok makes sense - I was wondering about that discrepancy so thanks for clarifying